docs/docs/01-ibc/06-channel-upgrades.md

```diff
@@ -136,6 +136,11 @@ The state will change to `FLUSHCOMPLETE` once there are no in-flight packets lef
 
 All other parameters will remain the same during the upgrade handshake until the upgrade handshake completes. When the channel is reset to `OPEN` on a successful upgrade handshake, the relevant fields on the channel end will be switched over to the `UpgradeFields` specified in the upgrade.
 
+> NOTE: due to the addition of new channel states, packets can still be received and processed in both `FLUSHING` and `FLUSHCOMPLETE` states.
+> Packets can also be acknowledged in the  `FLUSHING` state. Application developers should consider these new states
+> when implementing application logic that relies on the channel state.
+> It is still only possible to send packets when the channel is in the `OPEN` state.
```

Comment:
Should we use [admonitions](https://docusaurus.io/docs/markdown-features/admonitions)